manage.py

```diff
@@ -46,9 +46,23 @@ def __call__(self, app, host, port, use_debugger, use_reloader,
         if use_reloader is None:
             use_reloader = use_debugger
 
+        extra_files = None
```

Comment:
None will throw an error when you try to append to it in line 60 - should be

    extra_files = []